Plugin: 云签到

**插件名称：**

云签到

**插件功能：**

基于云端的签到，所有插件使用者数据互通

**PyPI 项目名：**

nonebot-plugin-cloudsignx

**插件 import 包名：**

nonebot_plugin_cloudsignx

**插件项目仓库/主页链接：**

https://github.com/Monarchdos/nonebot_plugin_cloudsign

**标签：**

[{"label":"a:云签到","color":"#2be491"},{"label":"a:数据互通","color":"#3be42b"}]

# 📃 商店发布检查结果

> Plugin: 云签到

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-cloudsign 的信息如下：
    name         : nonebot-plugin-cloudsign 
     version      : 1.0                      
     description  : Cloud sign in            
    
    dependencies
     - nonebot-adapter-onebot &gt;=2.2.0,&lt;3.0.0
     - nonebot2 &gt;=2.0.0-rc.2,&lt;3.0.0
插件 nonebot_plugin_cloudsign 加载出错：
01-05 03:59:36 [SUCCESS] nonebot | NoneBot is initializing...
01-05 03:59:36 [INFO] nonebot | Current Env: prod
01-05 03:59:36 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_cloudsign&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_cloudsign&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_cloudsign/__init__.py&quot;, line 10, in &lt;module&gt;
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
ModuleNotFoundError: No module named &#x27;requests&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: a:云签到-#2be491, a:数据互通-#3be42b。</li><li>✅ 项目 <a href="https://github.com/Monarchdos/nonebot_plugin_cloudsign">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-cloudsign/">nonebot-plugin-cloudsign</a> 已发布至 PyPI。</li></code></pre></details>

---

💪 Powered by NoneBot2 Publish Bot
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -3921,5 +3921,24 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_cloudsignx",
+    "project_link": "nonebot-plugin-cloudsignx",
+    "name": "云签到",
+    "desc": "基于云端的签到，所有插件使用者数据互通",
+    "author": "Monarchdos",
+    "homepage": "https://github.com/Monarchdos/nonebot_plugin_cloudsign",
+    "tags": [
+      {
+        "label": "a:云签到",
+        "color": "#2be491"
+      },
+      {
+        "label": "a:数据互通",
+        "color": "#3be42b"
+      }
+    ],
+    "is_official": false
   }
 ]

